bokeh/models/layouts.py

```diff
@@ -115,10 +115,12 @@ class LayoutDOM(Model):
     """)
 
     margin = Tuple(Int, Int, Int, Int, default=(0, 0, 0, 0), help="""
-    Allows to create additional space around the component.    
+    Allows to create additional space around the component.
+    
     The values in the tuple are ordered as follows - Margin-Top, Margin-Right, Margin-Bottom and Margin-Left,
     similar to CSS standards.
-    Negative margins could be added too to shrink the space from any direction.
+   
+    Negative margin values may be used too to shrink the space from any direction.
```

Comment:
typo: "too" -> "to"

Reply:
I actually meant that negative values may also be used. I'll remove the 'too'

Reply:
Oh I missed the other "to" that's easy to misread. FWIW I think the "too" can just be removed altogether and it will read clearly.